test/support/factory.ex

```diff
@@ -2,4 +2,6 @@ defmodule Liveman.Factory do
   use ExMachina.Ecto, repo: Liveman.Repo
 
   use Liveman.User.UserFactory
+  # credo:disable-for-next-line Credo.Check.Consistency.MultiAliasImportRequireUse
```

Comment:
I think we can disable this, 🤔  Maybe we should raise in the Elixir template cause we mostly do multi alias. Not sure maybe we can recheck with other Elixir projects 🤔  - https://github.com/rrrene/credo/blob/master/.credo.exs#L169

Reply:
I disabled the config for now. It mostly happens initial changes of the project but later when we use more multi alias, then `credo` does not show the warning. The Elixir projects(mostly internal) I can access, we did not disable the config nor comment in the line.

I will check it more and will raise issue in the template